tests/expose_test.go

```diff
@@ -739,7 +739,7 @@ var _ = Describe("[rfe_id:253][crit:medium][vendor:cnv-qe@redhat.com][level:comp
 func generateBatchJobSpec(host string, port string, clientBuilder func(host string, port string, checkConnectivityCmdPrefixes ...string) *batchv1.Job) *batchv1.Job {
 	if net.IsIPv6String(host) {
 		// TODO - remove this if condition code once https://github.com/kubevirt/kubevirt/issues/4428 is fixed
-		return clientBuilder(host, port, fmt.Sprintf("ping -c1 %s;", host))
+		return clientBuilder(host, port, fmt.Sprintf("ping -w 20 %s;", host))
```

Comment:
Please explain why this is needed in the commit msg; as it is, I don't understand why this was changed.

Reply:
I got no clue why `kubevirt-bot` is listed as this patch's author, but that's not good.

Reply:
Should be fixed now, including the commit message. `ping -c1` was usually failing, it starts working after the 3rd attempt. Instead of waiting for a successful single ping now we wait for a specific, sufficiently long amount of time for the host to be responding.

Reply:
Fair enough. Thanks.